File: smp2017-web-Darshit/CS50_Solutions/pset1/mario.c
#include<stdio.h>
#include<cs50.h>

int main()
{
    int height;
    do
    {
       printf("Height: ");
       height=GetInt();
    }
    while(height<0 || height>23);
    
    int i,space,symbol;
    for(i=height-1;i>=0;i--)
    {
        space=i;
        while(space>0)
        {
            printf(" ");
            space--;
        }
        symbol=height-i+1;
        while(symbol>0)
        {
            printf("#");
            symbol--;
        }
        printf("\n");
}
}

File: smp2017-web-Darshit/CS50_Solutions/pset6/mario.py
import cs50

print("Height: ", end="")
height = cs50.get_int()
while height < 0 or height > 23:
    print("Height: ", end="")
    height = cs50.get_int()

for i in range(height):
    space = height - i - 1
    while space > 0:
        print(" ", end="")
        space-=1
    
    symbol = i+2
    while symbol > 0:
        print("#", end="")
        symbol-=1
    
    print()
    
    